include/nrf_rpc/nrf_rpc_uart.h

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef NRF_RPC_UART_H_
+#define NRF_RPC_UART_H_
+
+#include <nrf_rpc.h>
+#include <nrf_rpc_tr.h>
+#include <zephyr/kernel.h>
+#include <zephyr/device.h>
+#include <zephyr/sys/ring_buffer.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @defgroup nrf_rpc_uart nRF RPC UART transport
+ * @brief nRF RPC UART transport.
+ *
+ * @{
+ */
+
+#define NRF_RPC_MAX_FRAME_SIZE	1536
+#define NRF_RPC_RX_RINGBUF_SIZE 2048
+
+typedef enum {
+	hdlc_state_unsync,
+	hdlc_state_frame_start,
+	hdlc_state_frame_found,
+	hdlc_state_escape,
+} hdlc_state_t;
+
+typedef struct nrf_rpc_uart {
```

Comment:
Shouldn't each member of the structure be documented with Doxygen tags, since this is a public API?